pages/api/rooms/create.js

```diff
@@ -23,9 +24,9 @@ client.interceptors.response.use(
 );
 
 
-export async function GenerateRoom() {
-  const response = await client.get('https://videolink2me.com/create_room');
-  return response.data;
+export async function GenerateRoomID() {
+  const buf = crypto.randomBytes(20);
+  return buf.toString('utf8');
```

Comment:
hex is better utf8 will have out of bound characters

Reply:
fixed in eab401b